docs/installation/tenants.asciidoc

```diff
@@ -0,0 +1,60 @@
+PacketFence supports multi-tenants and is by default in a mono-tenant mode.
+
+=== General concepts
+
+==== Built-in tenants
+
+Tenant `1` is the default tenant. A default PacketFence installation will use this tenant transparently.
+
+Tenant `0` is the global tenant. In a multi-tenant setup, you can modify
+configuration only in that tenant.
+
+All items configuration are global and share between all tenants. If you want
+to manage a multi-tenant setup, you need to deploy a specific configuration
+that will rely on tenant IDs.
+
+==== Domain name and portal domain name of a tenant
+
+A tenant can be created with two *optional* parameters:
+
+* a domain name: to create users in database using following convention: `username@DOMAIN_NAME_OF_TENANT`
+* a portal domain name: to assign nodes to a specific tenant when they reach PacketFence on this captive portal address
+
+These two parameters are used by code to assign tenants to PacketFence objects.
+
+When possible, we highly recommend you to create tenant with these two parameters.
+
+==== Network devices
+
+A network device is attached to one tenant at creation.
+
+When PacketFence receives a RADIUS request from a network device, it will
+instantiate a network device in this tenant and create node in this tenant.
+Then you can use `Tenant ID` field in connection profiles filters to apply
+specific configuration for each tenant.
+
+In some WebAuth scenarios, PacketFence doesn't receive RADIUS requests from
+network devices (only HTTP/s requests). Consequently, it can't assign tenant
+ID to nodes based on network devices' tenant ID. In that case, you need to
+defined a portal domain name in your tenant and filter on `FQDN` field in
+connection profiles.
+
+=== Getting started
+
+==== Creating your tenant
+
+Following command will create a new tenant, mapped to `example.com` domain and
+with a specific captive portal address `portal.example.com`:
+
+[source,bash]
+----
+/usr/local/pf/bin/pfcmd tenant add example example.com portal.example.com
```

Comment:
How do these instructions work when using a cluster? I guess packetfence-config needs to be restarted on all cluster member, as well as PacketFence itself.